### DISCUSSION TITLE
Finances API / List Financial Events / ServiceFeeEventList

### DISCUSSION THREAD

**Discussion by albertocattan** [amzn/selling-partner-api-models#3266] [General]
Hello!
I am successfully getting the data I need from the Finances API. However, the events that listed under the "**ServiceFeeEventList**" type seem to have no **PostedDate**, which all other event types do have... Am I missing something conceptually? Why don't these events need to show a posted date? Or is it a bug?
Thanks in advance,
Albert C

**Comment by takeno-dev** [comment]
I think that is the problem too.
I want to get these Date so I am running the process in periods of 1 day each. It is very annoying.
